pom.xml

```diff
@@ -19,6 +19,9 @@
         <distribution>repo</distribution>
         </license>
     </licenses>
+    <properties>
+        <jackson.version>[2.5.0, 2.5.5]</jackson.version>
```

Comment:
Should be `[2.5.0, 2.6)`

Reply:
This would take snapshot versions and release candidates, there is no way in maven to disable getting release candidates. This would only pick release versions

Reply:
@dmitraver,

How about using `[2.5.0, 2.6)` along with `<snapshots><enabled>false</enabled></snapshots>` in the `<repository>` tag?

Reply:
@thinkingserious yeah, I’ve tried disabling it that way but then it picks rc5 version which is not good and there is no way to workaround it afaik. Btw, the most recent version of Jackson is 2.9 something.